### ISSUE TITLE
Multiple build issues after upgrading to Xcode 15, Content and Service extensions missing Plist values

### ISSUE THREAD

**Issue by mobikats** [exponea/exponea-ios-sdk#51]
My project using the Exponea SDK has run into a lot of problems due to the extensions.

I've worked through several - missing keys in the Info.plist mainly, but now am struggling to get past this:

Asset validation failed
Invalid bundle structure. The “x.app/PlugIns/ContentExtension.appex/ContentExtension” binary file is not permitted. Your app cannot contain standalone executables or libraries, other than a valid CFBundleExecutable of supported bundles. For details, visit: https://developer.apple.com/documentation/bundleresources/placing_content_in_a_bundle (ID: c4d774ab-7203-4dab-9e1d-cfba0e823eae)

For both the ContentExtension and ServiceExtension.

It was difficult to even find out that these were the problem originally because all that happened was that I could not install the app for Ad Hoc testing, but eventually I ended up here.

Perhaps someone else recently resolved this and got further than I have :)

**Comment by Ankmara** [comment]
Hi, thank you for contacting us. We didn't support Xcode yet, but I have few questions.
1) are you using source code from GitHub or cocoapods/carthage/spm?
2) did you have any issue with Xcode (save and revert dialog)?
3) What did you do after installation Xcode 15?

Thank you for response.

**Comment by xcibik00** [comment]
Just note that Xcode 15 support is in our new SDK v2.19.0

**Comment by xcibik00** [comment]
Hi @mobikats, is it working for you correctly on newer iOS  SDK versions 2.19.0+? (with support of Xcode15)

**Comment by mobikats** [comment]
Hi sorry everyone for some reason I didn't reply earlier. I forget how I resolved this now, I am using it within a Flutter project, and the version is listed as 1.4.2 there (I don't know what the iOS-specific version it is wrapping):
  exponea:
    dependency: "direct main"
    description:
      name: exponea
      url: "https://pub.dev"
    source: hosted
    version: "1.4.2"

This is building and running and I believe functions correctly.
I expect I solved the missing plist entries manually and carried on.
